### ISSUE TITLE
Freeze or Black Screen with Mega Moemon 1.1.4a

### ISSUE THREAD

**Issue by LazyBunny** [visualboyadvance-m/visualboyadvance-m#393]
### Expected behavior
I expect to be able to access a pokemart shop with the Mega Bracelet or mega-evolution poke-rite stones as i would a normal poke mart. And if i already have the Mega Bracelet/stones, i expect to be able select them in the inventory.

### Actual behavior
If i attempt to open a vender selling the bracelet or mega evolution stones, i get a black screen, and the FPS will often drop to about 7 or so.

If i already have the bracelet or stones, from adding them using another emulator, and i attempt to select them in the Bag/Inventory list. The game Freezes just before getting to those items. The emulator's menu's still function.
### Steps to reproduce the behavior
Load the Pokemon FireRed Rom-hack, Mega Moemon 1.1.4a, advance to Celadon city, goto the 4th floor i believe, and talk to the first vender, and the screen will go black.

### ROM details
Mega Moemon 1.1.4a, Rom Hack of Pokemon Fire Red US.
https://forums.pokemmo.eu/index.php?/topic/71562-ips-mega-moemon-firered-2019-w-new-emerald/

Please do not dismiss this because it's a rom-hack. I'm not reporting this here simply because it's freezing/blackscreen in VBA-M.. i'm reporting it because it's NOT freezing/blackscreening on VBA 1.8.0beta3

### Option details
Video driver (simple, opengl): OpenGL, Simple 4x
Sound driver (directsound, xaudio, openal, SDL): SDL

### Build details
Operating System:
Windows 10 Pro 1803 x64

Version of code: I'm not sure what this means.. so i'll say VBA-m 2.1.0

### Hardware details:
GPD Win2, 
Core m3 7y30 @ 2.5ghz
8GB Ram
Intel HD 615. I don't remember the driver version.
256GB M.2 ADATA 2242 SATA-III SSD. Game and Emulator running from 128GB USB 3.0 Thumb Drive.

### Additional Details
This rom hack does work on VBA 1.8.0beta3 (AKA VBA-OG), without crashing, but that emulator is dated and lacks certain features that i enjoy in VBA-M. 
I thought i could get past the Black-Screen of the PokeMark vender by loading the save/rom into VBA-OG, buying the bracelet and evolution stones, and then going back into VBA-M.
But that's when i discovered the Freeze Menu. when attempting to scroll down to the bracelet.
Mega Moemon is a very popular Romhack and breaths new life into some rather old classics. As it brings in features and mechanics of newer games, back into the classics.
So I would very much so like to see this fixed. So please don't dismiss it because it's a rom-hack. Pokemon Rom-Hacks are more amazing now than ever before.

**Comment by LazyBunny** [comment]
### And... i think i might have solved my own problem...
I feel both silly and annoyed.
I always used the real bios file out of habit. For years, when ever i'd use any GBA emulator, i'd use a GBA bios, just for compatibility. This time i turned it off, just to see what would happen... and sure enough.. it seems to be working.

**Comment by retro-wertz** [comment]
alot of these Pokemon hacks does not work with real hardware. im not surprised that would not work when using with real bios file. this is why most of them dont patches available in RomHacking.net and mostly includes a rom download, which is illegal...

**Comment by LazyBunny** [comment]
Yeah, that's true, though oddly enough, i found a lot of rom hacks DO work amazing on original hardware. I actually want some moemon repro carts :D Same with pokemon Prism
